internal/image/ova.go

```diff
@@ -179,6 +187,20 @@ func (o *Ova) ovfSize() (int, error) {
 	return calculateTarSize(int(fileInfo.Size())), nil
 }
 
+func (o *Ova) diskSize() (int, error) {
+	file, err := os.Open("data/persistence-disk.vmdk")
```

Comment:
I just wonder if we really need to bring the disk here, could not user create it when deploying OVA?

Reply:
Can't we just leave the option for the user? if he decides to add a disk we move `data` folder there if not we proceed as planned?

Reply:
I am not sure what the benefit will be? currently it is a lazy provisioned very small file, it is now part of the OVA and will be deployed on the same storage selected by the user in the process of installation.
There is a higher complexity in allowing the user to add a disk - we do not know if it is sata or iscsi, not sure of it's id or name. What if the user wants to add a disk for a different purpose and wipes it so we loose the credentials and the id? Do we require that he adds a specific label? what if they remove the disk later (it is their disk...).
I think, at least for persistence of critical elements that like credentials this is a better solution, but I am open to be convinced.

Reply:
Yep, makes sense. I wasn't just sure if in the process of deployment user still need to select the disk for another VMDK, but I understand he don't have to and he pick the storage just once for both, which is great. So let's do it your way for sure.

Reply:
I wasn't sure how the deployment process will be. If we automate the disk it's fine then.

just a remark: we never lose the id. the agent_id is created when agent service starts.